src/ripple/app/rdb/backend/RelationalDBInterfaceSqlite.cpp

```diff
@@ -1315,6 +1386,10 @@ RelationalDBInterfaceSqliteImp::getTransaction(
     std::optional<ClosedInterval<uint32_t>> const& range,
     error_code_i& ec)
 {
+    /* if Tx tables not used, return empty value */
```

Comment:
Comment isn't quite right. It returns "unknown", not an empty value.

Reply:
Done